aws/config_recorder.go

```diff
@@ -0,0 +1,83 @@
+package aws
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/configservice"
+	"github.com/gruntwork-io/cloud-nuke/config"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/cloud-nuke/report"
+	"github.com/gruntwork-io/go-commons/errors"
+)
+
+func getAllConfigRecorders(session *session.Session, excludeAfter time.Time, configObj config.Config) ([]string, error) {
+	svc := configservice.New(session)
+
+	configRecorderNames := []string{}
+
+	param := &configservice.DescribeConfigurationRecordersInput{}
+
+	output, err := svc.DescribeConfigurationRecorders(param)
+	if err != nil {
+		return []string{}, errors.WithStackTrace(err)
+	}
+
+	for _, configRecorder := range output.ConfigurationRecorders {
+		if shouldIncludeConfigRecorder(configRecorder, excludeAfter, configObj) {
+			configRecorderNames = append(configRecorderNames, aws.StringValue(configRecorder.Name))
+		}
+	}
+
+	return configRecorderNames, nil
+}
+
+func shouldIncludeConfigRecorder(configRecorder *configservice.ConfigurationRecorder, excludeAfter time.Time, configObj config.Config) bool {
+	if configRecorder == nil {
+		return false
+	}
+
+	return config.ShouldInclude(
+		aws.StringValue(configRecorder.Name),
+		configObj.ConfigServiceRecorder.IncludeRule.NamesRegExp,
+		configObj.ConfigServiceRecorder.ExcludeRule.NamesRegExp,
+	)
+}
+
+func nukeAllConfigRecorders(session *session.Session, configRecorderNames []string) error {
```

Comment:
Function return `nil` as error each time, make sense to have error in the signature?

Reply:
Probably not, but in order to satisfy the interface for `AwsResource` it needs to :/ I'll think about how to clean this up.